hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/HealthTransformations.scala

```diff
@@ -68,8 +67,42 @@ object HealthTransformations {
       None
     }
 
+  def mapEyeDisease(rawRecord: RawRecord): Iterable[HlesHealthCondition] =
+    if (rawRecord.getBoolean("hs_dx_eye_yn")) {
+      eye_diseases.flatMap {
+        // "other" case
+        case ("eye_other", categorical) =>
+          createHealthConditionRow(
+            rawRecord,
+            "dx_eye_other",
+            eyeDiseaseCondition,
+            categorical,
+            conditionOtherDescription = rawRecord.getOptional("hs_dx_eye_other_spec")
+          )
+        case ("blind", categorical) => {
+          val isCauseKnown = rawRecord.getBoolean("hs_dx_eye_cause_yn")
+          val conditionCause = if (isCauseKnown) { rawRecord.getOptionalNumber("hs_dx_eye_cause") }
+          else { None }
```

Comment:
Wot is dis scalafmtery

Better would be either:
```scala
val conditionCause =
  if (isCauseKnown) rawRecord.getOptionalNumber("hs_dx_eye_cause") else None
```
Or
```scala
val conditionCause = if (isCauseKnown) {
  rawRecord.getOptionalNumber("hs_dx_eye_cause")
} else {
  None
}
```
Same for the other if/else clauses in this file, can you update them?